hack/update-mockgen.sh

```diff
@@ -87,3 +87,20 @@ done
 
 goimports -w "$beacon_api_mock_path/."
 gofmt -s -w "$beacon_api_mock_path/."
+
+# github.com/prysmaticlabs/prysm/v3/crypto/bls
+# --------------------------------------------
+crypto_bls_common_mock_path="crypto/bls/common/mock"
+crypto_bls_common_mocks=(
+      "$crypto_bls_common_mock_path/interface_mock.go interface.go"
```

Comment:
are you using this somewhere in the tests right now? having trouble connecting these changes to the rest of the PR.